<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>757</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Algorithm Ramer-Douglas-Peucker </string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Pictures/RDP_str.png</normaloff>
    <normalon>../../Pictures/RDP_str.png</normalon>
    <disabledoff>../../Pictures/RDP_str.png</disabledoff>
    <disabledon>../../Pictures/RDP_str.png</disabledon>
    <activeoff>../../Pictures/RDP_str.png</activeoff>
    <activeon>../../Pictures/RDP_str.png</activeon>
    <selectedoff>../../Pictures/RDP_str.png</selectedoff>
    <selectedon>../../Pictures/RDP_str.png</selectedon>../../Pictures/RDP_str.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(71, 147, 170);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>601</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_vvedite_2">
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(149, 160, 255, 255), stop:1 rgba(164, 231, 174, 255));
</string>
       </property>
       <property name="text">
        <string>Постройте исходную кривую с помощью ЛКМ в области графика</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MplWidget" name="MplWidget" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_generate">
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(198, 221, 255);</string>
       </property>
       <property name="text">
        <string>Упростить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_save">
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(198, 221, 255);</string>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_clear">
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(198, 221, 255);</string>
       </property>
       <property name="text">
        <string>Сбросить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>600</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLineEdit" name="line_x">
           <property name="sizeIncrement">
            <size>
             <width>4</width>
             <height>4</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
background-color: rgb(255, 255, 255);
gridline-color: rgb(255, 0, 255);
border-color: rgb(0, 170, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_x">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(149, 160, 255, 255), stop:1 rgba(164, 231, 174, 255));
</string>
           </property>
           <property name="text">
            <string>x</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLineEdit" name="line_y">
           <property name="sizeIncrement">
            <size>
             <width>5</width>
             <height>5</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
gridline-color: rgb(255, 0, 255);
border-color: rgb(0, 170, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_y">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(149, 160, 255, 255), stop:1 rgba(164, 231, 174, 255));
</string>
           </property>
           <property name="text">
            <string> у</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(198, 221, 255);</string>
       </property>
       <property name="text">
        <string>Изменить область координат</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_vvedite">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(149, 160, 255, 255), stop:1 rgba(164, 231, 174, 255));
</string>
    </property>
    <property name="text">
     <string>Введите область координат</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_e">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>219</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_e">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(149, 160, 255, 255), stop:1 rgba(164, 231, 174, 255));
</string>
    </property>
    <property name="text">
     <string>Epsilon:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
